### ISSUE TITLE
guard against sql injection

### ISSUE THREAD

**Issue by talwinder50** [hyperledger/aries-framework-go#1986]

**Comment by llorllale** [comment]
@talwinder50 didn't you say some of this is inevitable due to https://github.com/go-sql-driver/mysql/issues/848 ?

**Comment by talwinder50** [comment]
Yes warning will not go away, its just precautionary step i can add that is escape characters.
